chart/flux/CHANGELOG.md

```diff
@@ -1,3 +1,17 @@
+## 1.6.1 (2021-02-04)
+
+### Improvements
+
+ - Updated Flux to `1.21.1`
+   [fluxcd/flux#3393](https://github.com/fluxcd/flux/pull/3393)
+ - snap: move to core20 [fluxcd/flux#3371](https://github.com/fluxcd/flux/pull/3371)
```

Comment:
We track here the Helm chart changes not snap. Please remove all changes that have no affect on the Helm chart.

Reply:
OK. fixed